projects/ion/src/lib/message/message.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { CommonModule } from '@angular/common';
+import { RenderResult, render, screen } from '@testing-library/angular';
+
+import { IonMessageComponent } from './message.component';
+import { IonMessageProps, MessageStatusType } from './types';
+import { IonIconComponent } from '../icon';
+
+const defaultValue: IonMessageProps = {
+  label: 'Message',
+};
+
+export const icontypes = [
```

Comment:
```suggestion
export const icontypes: MessageStatusType[] = [
```
if you type it here you wont have to do the type assertion.